k8s/upgrades/0.8.0-0.8.1/cstor_volume_upgrade.sh

```diff
@@ -64,29 +67,63 @@ sc_res_ver=`kubectl get sc $sc_name -n $sc_ns -o jsonpath="{.metadata.resourceVe
 # c-dep: pvc-cec8e86d-0bcc-11e8-be1c-000c298ff5fc-target        # 
 #################################################################
 
-c_dep=$(echo $pv-target)
-c_svc=$(echo $pv)
-c_vol=$(echo $pv)
+c_dep=$(kubectl get deploy -n $ns -l openebs.io/persistent-volume=$pv,openebs.io/target=cstor-target -o jsonpath="{.items[*].metadata.name}")
+c_svc=$(kubectl get svc -n $ns -l openebs.io/persistent-volume=$pv,openebs.io/target-service=cstor-target-svc -o jsonpath="{.items[*].metadata.name}")
+c_vol=$(kubectl get cstorvolumes -l openebs.io/persistent-volume=$pv -n $ns -o jsonpath="{.items[*].metadata.name}")
+c_replicas=$(kubectl get cvr -n openebs -l cstorvolume.openebs.io/name=$pv -o jsonpath="{range .items[*]}{@.metadata.name};{end}" | tr ";" "\n")
```

Comment:
here should we be using "c_vol" rather than "pv"?

Reply:
yeah updated it